packages/fluentui/react-bindings/src/hooks/useStyles.ts

```diff
@@ -113,11 +113,11 @@ const useStyles = <StyleProps extends PrimitiveProps>(
     allDisplayNames: composeOptions?.displayNames || [displayName],
     className: composeOptions?.className || className,
     primaryDisplayName: composeOptions?.displayName || displayName,
-    props: {
+    componentProps: {
       ...componentStylesProps,
-      ...mapPropsToInlineStyles(),
       ...composeStylesProps,
     },
+    stylesProps: mapPropsToInlineStyles(),
```

Comment:
Maybe `inlineStylesProps` would be better name?

Reply:
Agree, will rename